### ISSUE TITLE
qt-5.10

### ISSUE THREAD

**Issue by udeved** [artix-linux/world#44]
https://bugreports.qt.io/browse/QTBUG-64490

How we gonna handle this?

We have lts kernel running on server.

arch patch qt5-base...
~~~
# Don't use the statx syscall https://bugs.archlinux.org/task/56289 https://bugreports.qt.io/browse/QTBUG-64490
  patch -p1 -i ../qt-no-statx.patch
~~~

**Comment by udeved** [comment]
Not sure about this one. @graysky2 @cromerc 
https://bugs.archlinux.org/task/56289

**Comment by cromerc** [comment]
My recommendation would be rebase lts to 4.14. 4.14 is the next official lts kernel.

**Comment by udeved** [comment]
I tend to agree @cromerc 

Not sure its best path for us to patch out statx due to systemd breakage.

**Comment by udeved** [comment]
These arch packages produce breakage if arch testing is enabled.

* plasma-workspace-5.11.4-2
* plasma-wayland-session-5.11.4-2
* plasma-integration-5.11.4-2
* plasma-framework-5.40.0-2
* kwin-5.11.4-2
* kuiserver-5.11.4-2
* kdeclarative-5.40.0-2

**Comment by udeved** [comment]
I think we need to build with arch patches for now. We can't wait until January for new lts, and installing linux-4.14 from arch core would need server reboot and us adding it to system repo.

**Comment by udeved** [comment]
Finished with a little cheating, I pulled arch binary webengine and webkit in our repo and resigned them.

**Comment by physkets** [comment]
@udeved Isn't the issue only if one of the two conditions occurs:

* The build machine has systemd
* The build machine has linux>=4.11

From what you've said, we have neither of those issues, and we should be able to complie without the patch, no? If so, why is the jenkins build failing?

**Comment by udeved** [comment]
@physkets 

No we are not able to compile without patches since thne build kernel is lts.
The lts krnel would have to be rebased to 4.14 to have no statx feature.

So everything fine for now, we compiled without statx.

**Comment by physkets** [comment]
Ah! So the statx() syscall is only available in and after 4.11; I hadn't understood that.
